docs/setup.md

```diff
@@ -3,31 +3,21 @@
 
 # Installation
 
+> [!NOTE]  
+> The **Fortinet AI Assistant** widget installs with the [*FortiAI* solution pack](https://github.com/fortinet-fortisoar/solution-pack-fortinet-advisor/blob/develop/docs/setup.md#installation) and hence may not require separate installation.
+
 1. Click **Content Hub** > **Discover**.
 
 2. Search **AI Assistant** in the list of widgets that appears. 
 
 3. Click the **AI Assistant** widget card.
 
-4. Click **Install** on the lower part of the screen to begin installation.
-
-> [!NOTE]  
-> The **FortiAI Configuration** widget installs with the [*FortiAI* solution pack](https://github.com/fortinet-fortisoar/solution-pack-fortinet-advisor/blob/develop/docs/setup.md#installation) and hence may not require separate installation.
-
 # Configuration
 
 > [!NOTE]  
```

Comment:
but it needs FortiAI configuration before user starts using it

Reply:
User uses the FortiAI SP not the bot. That distinction, between the SP and widgets, is internal.